### PR TITLE
optionally load the tasks from yast-rake-ci if it is installed

### PR DESCRIPTION
**Pull request by lslezak**
- 0.2.3

### CODE REVIEW

File: lib/yast/rake.rb [hunk 1]
```diff
@@ -56,3 +56,10 @@ def self.spec_version
   load f
 end
 
+# optionally load the tasks from yast-rake-ci if it is installed
+begin
+  require "yast/rake_ci"
+  rescue LoadError
```
Comment (jreidinger) [inline lib/yast/rake.rb]: wrong indentation here

**Comment by jreidinger** [comment]
otherwise looks good.

File: lib/yast/rake.rb (hunk 1 above)
Comment (lslezak) [inline lib/yast/rake.rb]: I see, we do not use `rubocop` here... :wink:

**Comment by jreidinger** [comment]
LGTM

**Pull request merged**